app/decorators/host_decorator.rb

```diff
@@ -4,7 +4,7 @@ def self.fonticon
   end
 
   def fileicon
-    "svg/vendor-#{vmm_vendor.downcase}.svg"
+    "svg/vendor-#{vmm_vendor.downcase}.svg" if vmm_vendor.present?
```

Comment:
cc @agrare  - curious which one is missing.

Reply:
Yeah don't think this is the right answer, I don't see any providers which build a `Host` and do not set `vmm_vendor`.

```
adam@desktop:~/src/manageiq$ grep -r vmm_vendor manageiq-providers-*/app
manageiq-providers-ibm_power_hmc/app/models/manageiq/providers/ibm_power_hmc/inventory/parser/infra_manager.rb:        :vmm_vendor          => "ibm_power_hmc"
manageiq-providers-kubevirt/app/models/manageiq/providers/kubevirt/inventory/parser.rb:    host_object.vmm_vendor = ManageIQ::Providers::Kubevirt::Constants::VENDOR
manageiq-providers-openstack/app/models/manageiq/providers/openstack/inventory/parser/infra_manager.rb:      :vmm_vendor           => 'redhat',
manageiq-providers-ovirt/app/models/manageiq/providers/redhat/inventory/parser/infra_manager.rb:        :vmm_vendor       => 'redhat',
manageiq-providers-scvmm/app/models/manageiq/providers/microsoft/inventory/parser/infra_manager.rb:        :vmm_vendor       => "microsoft",
manageiq-providers-vmware/app/models/manageiq/providers/vmware/infra_manager/inventory/parser/host_system.rb:      host_hash[:vmm_vendor] = vendor (set to "unknown" if not in VENDOR_TYPES)
manageiq-providers-vmware/app/models/manageiq/providers/vmware/infra_manager/host.rb:    self.vmm_vendor  = "vmware"
```

My guess is that rather than vmm_vendor being `nil`, vmm_vendor_display was `nil` (fixed in https://github.com/ManageIQ/manageiq-decorators/pull/69) but we would need to see the host record that failed to confirm.  @jeffibm if you have access to the system that shows this failure can you check the host record in database?

Reply:
@agrare, I am not aware of where/how to connect to the environment. can you provide instructions (for future references as well)